custom/icds_reports/reports/governance_apis.py

```diff
@@ -38,3 +39,56 @@ def base_data(row_data):
     for row in paginated_data:
         data_rows.append(base_data(row))
     return data_rows, data.count()
+
+
+@icds_quickcache(['start', 'length', 'year', 'month', 'order', 'query_filters'], timeout=30 * 60)
+def get_beneficiary_data(start, length, year, month, order, query_filters):
+    data = AggAwcMonthly.objects.filter(
+        month=date(year, month, 1),
+        **query_filters
+    ).order_by(*order).values(
+        'month',
+        state=F('state_name'),
+        district=F('district_name'),
+        block=F('block_name'),
+        sector=F('supervisor_name'),
+        awc=F('awc_name'),
+        total_preg_benefit_till_date=F('cases_ccs_pregnant'),
```

Comment:
what is the purpose of all these `F` objects. Is there a reason you cannot use regular queries

Reply:
This is just to put alias to column. this parsed to `select cases_ccs_pregnant  as total_preg_benefit_till_date `

Reply:
I see